### PR TITLE
Refactor to output both ESM and IIFE targets

### PR DESCRIPTION
**Pull request by barrymcgee**
Note: Due to the updated function call signature, this is a breaking change.

## Done

- Updated package to export both ESM nd IIFE targets

## QA

- Pull down this feature branch
- Open `index.html` in a browser
- Use `npm link` to check this package can be consumed by JS application (e.g. use create-react-app to spin up sample app, then `npm i` it)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,16 +6,40 @@ You can use it to display a custom cookies message of your choosing on your own
 
 ## Usage
 
-This project can be installed from an [NPM package](https://www.npmjs.com/package/cookie-policy).
+This project can be installed via an [NPM package](https://www.npmjs.com/package/cookie-policy).
+
+```bash
+yarn add cookie-policy
+npm install cookie-policy --save
```
Comment (hatched) [inline README.md]: `--save` is the default now

File: README.md [hunk 2]
```diff
@@ -6,16 +6,40 @@ You can use it to display a custom cookies message of your choosing on your own
 
 ## Usage
 
-This project can be installed from an [NPM package](https://www.npmjs.com/package/cookie-policy).
+This project can be installed via an [NPM package](https://www.npmjs.com/package/cookie-policy).
+
+```bash
+yarn add cookie-policy
```
Comment (hatched) [inline README.md:12]: maybe add an `# or` between these two lines.

File: README.md [hunk 3]
```diff
@@ -6,16 +6,40 @@ You can use it to display a custom cookies message of your choosing on your own
...
+```
+
+You can then install the library either by directly linking to it or via an ES6 import.
+
+1. Via direct link
+
+To consume the library directly, add a link to the JS file containing an [IIFE](https://developer.mozilla.org/en-US/docs/Glossary/IIFE) and call the lib:
+
+```javascript
+<script src="/node_modules/cookie-policy/build/js/cookie-policy.js"></script>
```
Comment (hatched) [inline README.md]: `node_modules` should be outside of the static paths of a webserver for security reasons so we should encourage people to copy the asset if they will not be importing the package directly.

File: bower.json [hunk 4]
```diff
@@ -2,17 +2,9 @@
   "name": "canonical-cookie-policy",
```
Comment (hatched) [inline bower.json]: Do we still require a bower config?

Comment (barrymcgee) [reply bower.json]: No, removing.

**Review by hatched — APPROVED** [review]

File: package.json [hunk 5]
```diff
@@ -2,20 +2,21 @@
   "name": "cookie-policy",
   "version": "1.1.0",
```
Comment (hatched) [inline package.json:3]: You should probably bump the major version for this change.

Comment (barrymcgee) [reply package.json:3]: That'll be a separate PR.

**Review by anthonydillon — APPROVED** [review]
LGTM :+1:

**Pull request merged**